lib/plugin_version.rb

```diff
@@ -80,10 +80,26 @@ def check_manifest_version_validity
   def check_targets_validity
     targets_names = @targets.map {|t| t["name"] }
 
-    if @manifest["targets"].nil? ||
-       @manifest["targets"].empty? ||
-       !@manifest["targets"].all? { |target| targets_names.include?(target) }
+    if @manifest["targets"].nil? || @manifest["targets"].empty?
+      platform_targets = {
```

Comment:
I would extract it to a const or a different method called something like `targets_mapper`